lib/tasks/data/workbench_upload.rb

```diff
@@ -1,69 +1,69 @@
 # :nocov:
-require 'aws-sdk-s3'
```

Comment:
Is there a reason we are using Typhoeus directly instead of using the S3 SDK? 

Also, what about uploading directly to google instead of using MinIO?